docs/src/admin-manual/dashboard-build.md

```diff
@@ -0,0 +1,215 @@
+# Building and serving the dashboard
+
+This page describes how to build and serve the Enduro dashboard. The dashboard
+is a [Vue.js] application built with [Vite] that provides the web interface
+for managing and monitoring Enduro.
+
+## Requirements
+
+To build the dashboard from source, you need:
+
+- **Node.js** (version 20 or later)
+- **npm** (included with [Node.js])
+
+## Environment
+
+The following environment variables are used to configure the dashboard:
+
+```txt
+VITE_OIDC_ENABLED
+VITE_OIDC_BASE_URL
+VITE_OIDC_AUTHORITY
+VITE_OIDC_CLIENT_ID
+VITE_OIDC_SCOPES
+VITE_OIDC_ABAC_ENABLED
+VITE_OIDC_ABAC_CLAIM_PATH
+VITE_OIDC_ABAC_CLAIM_PATH_SEPARATOR
+VITE_OIDC_ABAC_CLAIM_VALUE_PREFIX
+VITE_OIDC_ABAC_USE_ROLES
+VITE_OIDC_ABAC_ROLES_MAPPING
+
+VITE_INSTITUTION_LOGO
+VITE_INSTITUTION_NAME
+VITE_INSTITUTION_URL
+```
+
+Check the [Dashboard configuration](dashboard-config.md) for more information
+about these variables. When building from source, these variables will be read
+from the process environment or fallback to the values in the `.env` file.
```

Comment:
is "process environment" a technical term? If not, perhaps this should be "processed environment"?

Also, I would say: 

... or else will fall back to the values in the `.env` file.